tests/parm/fd_nems.yaml

```diff
@@ -1149,12 +1153,67 @@
        canonical_units: 1
        description: AOD
      #
+     - standard_name: Sl_sfrac
+       alias: mean_snow_area_fraction_lnd
+       canonical_units: 1
+       description: land export 
+     #
+     - standard_name: Sl_tref
+       alias: inst_temp_height2m_lnd
+       canonical_units: K
+       description: mediator export to glc - no levation classes
+     #
+     - standard_name: Sl_qref
+       alias: inst_spec_humid_height2m_lnd
+       canonical_units: kg kg-1
+       description: land export
+     #
+     - standard_name: Sl_q
+       alias: inst_spec_humid_lnd
+       canonical_units: kg kg-1
+       description: land export
+     #
      - standard_name: Fall_lat
```

Comment:
@uturuncoglu I am working on Issue #1800 and want to check here for the land component. In what sense are the fields from land to atm "mean" fields? 

Right now, the import fields to fv3ATM almost all use the term "mean" but I am trying to get away from that since it is very ambiguous what "mean" refers to. I would prefer that additional information be in the field description.

Reply:
@DeniseWorthen I think we are using instantaneous fields to force the component. I think for land component going with inst_* would be fine.

Reply:
So for the import field in atmos_model ``mean_sensi_heat_flx_lnd`` this could also be called ``inst_sensi_heat_flx_lnd`` and the fd_nems.yaml would be ``Fall_sen``.

Reply:
@DeniseWorthen Yes. That was just a convention used in the field dictionary. If `Fall_sen` is called as `inst_sensi_heat_flx_lnd` that will be fine for component land.